src/bundle/Resources/views/fieldtypes/ezkeyword.html.twig

```diff
@@ -0,0 +1,6 @@
+{%- block ezplatform_fieldtype_ezkeyword_widget -%}
+    <div class="ez-data-source__taggify form-control"></div>
+    <div class="ez-data-source__input-wrapper" hidden>
+        {{- block('form_widget') -}}
```

Comment:
Can't we just hide the input itself instead of creating a boilerplate `div`?

Reply:
It might be done in many ways. It's just one of them. What do you suggest instead?
Please, provide a code sample.

Reply:
To hide the input itself:
```
{%- block ezplatform_fieldtype_ezkeyword_widget -%}
    {% set attr = attr|merge({'hidden': 'hidden'}) %}
    <div class="ez-data-source__taggify form-control"></div>
    {{- block('form_widget') -}}
{%- endblock -%}
```